apps/src/templates/PaneHeader.jsx

```diff
@@ -162,8 +163,9 @@ export const PaneButton = Radium(function (props) {
     >
       <span style={styles.headerButtonSpan}>
         {props.hiddenImage}
-        <i className={props.iconClass} style={iconStyle}/>
+        {!props.isRtl && <i className={props.iconClass} style={iconStyle}/>}
```

Comment:
@Erin007 Do you know of a better way to do this?

Reply:
What you have is consistent with how we do this in other places, such as https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/LargeChevronLink.jsx.